model/fv_arrays.F90

```diff
@@ -671,6 +682,7 @@ module fv_arrays_mod
                               !< original value before entering the physics; a value of 0.7 roughly
                               !< causes the energy fixer to compensate for the amount of energy changed
                               !< by the physics in GFDL HiRAM or AM3.
+   real    :: tau_w = 0.      !< Time scale (in days) for Rayleigh friction applied to vertical wind
```

Comment:
I believe the description for tau_w and tau need to be updated.  If I'm reading the code correctly, when tau_w is defined as zero (the default), tau still controls the RF damping timescale for the vertical winds.  But if tau_w is anything other than zero, it is now and independent timescale for the RF damping of the vertical winds and tau no longer has any effect.  Also think there needs to be some discussion of what happens in a hydrostatic system.  Is this something that should only be used in WAM simulations or does it have general application?

Reply:
@bensonr @hmhjuang 
You are right. 
It should work for any general application. I believe it does not affect hydrostatic system. I will add more description.